templates/lobbys/lobby.html

```diff
@@ -177,10 +211,45 @@ <h2 v-else>Lobby [[lobbyInfo.lobby_id]]: </h2>
         <span id="custom-tooltip" ref="shareTooltip">Copied invite to clipboard!</span>
     </div>
 
+    
 
     <template v-if="lobbyInfo.user.owner">
+
         <hr>
-        <h4>Manage: Add a prompt</h4>
+
+        <h3>Lobby settings</h4>
+
+        <details>
+            <summary>List of players</summary>
+            <div class="card">
+                <div class="card-body">
+                    <table class="table table-hover">
+                        <thead>
+                            <th scope="col">Display name</th>
+                            <th scope="col">Host</th>
+                            <th scope="col">Full user</th>
```

Comment:
(personal preference)

I think "Full user" is kind of confusing too. I think "Registered User" or "Anonymous User" (and flipping the condition) would be more clear